manifests/q/qBittorrent/qBittorrent/4.6.4/qBittorrent.qBittorrent.locale.zh-CN.yaml

```diff
@@ -0,0 +1,42 @@
+# Created with YamlCreate.ps1 v2.3.4 Dumplings Mod $debug=QUSU.CRLF.7-4-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.6.0.schema.json
+
+PackageIdentifier: qBittorrent.qBittorrent
+PackageVersion: 4.6.4
+PackageLocale: zh-CN
+Publisher: The qBittorrent project
+PublisherUrl: https://www.qbittorrent.org/
+PublisherSupportUrl: https://github.com/qbittorrent/qBittorrent/issues
+# PrivacyUrl:
+Author: |-
+  Sledgehammer999 <sledgehammer999@qbittorrent.org>
+  Christophe Dumez <chris@qbittorrent.org>
+PackageName: qBittorrent
+PackageUrl: https://www.qbittorrent.org/
+License: GPL-3.0
+LicenseUrl: https://github.com/qbittorrent/qBittorrent/blob/master/COPYING
+Copyright: 版权所有 © 2006-2024 The qBittorrent project
+# CopyrightUrl:
+ShortDescription: 免费（自由）可靠的 P2P Bittorent 客户端
+Description: |-
+  qBittorrent 是一款使用 C++ / Qt 编写的 bittorrent 客户端，使用了 Arvid Norberg 的 libtorrent（有时也被称为 libtorrent-rasterbar）。
+  它旨在成为所有其它 bittorrent 客户端的良好替代品。qBittorrent 快速、稳定，并提供 unicode 支持等特性。
+# Moniker:
+Tags:
+- p2p
+- 上传
+- 下载
+- 共享
+- 分享
+- 磁力链接
+- 磁链
+- 种子
+# ReleaseNotes:

```

Comment:
v4.6.4 changelog:
- BUGFIX: Correctly adjust "Add New torrent" dialog position in all the cases (glassez)
- BUGFIX: Change "metadata received" stop condition behavior (glassez)
- BUGFIX: Add a small delay before processing the key input of search boxes (Chocobo1)
- BUGFIX: Ensure the profile path is pointing to a directory (Chocobo1)
- RSS: Use better icons for RSS articles (glassez)
- WINDOWS: NSIS: Update French, Hungarian translations (MarcDrieu, foxi69)
- LINUX: Fix sorting when ICU isn't used (Chocobo1)
- LINUX: Fix invisible tray icon on Plasma 6 (tehcneko)